components/desktop/KnitFooter.vue

```diff
@@ -1,13 +1,38 @@
 <template>
   <footer class="knit-footer">
-    Footer
+    <a href="/" class="knit-footer__link">
+      <img src="/knit-logotype.svg" class="knit-footer__logo" alt="Logo of Koło Naukowe IT">
```

Comment:
When an image is inside a link tag, an `alt` attribute should describe a function of that link, not what this image is. In this example, KNIT logotype links to homepage, not to KNIT logotype. In Polish website, alternative text should be written in polish language also. Please fix it to e.g. 'Koło Naukowe IT'. You can read this article too: https://webaim.org/techniques/alttext/

Reply:
@zrebcu411 Thanks bro, that link which you've provided is pure gold.